packages/fake-browser/src/apis/tabs.ts

```diff
@@ -155,6 +156,34 @@ export const tabs: BrowserOverrides['tabs'] = {
     await onHighlighted.trigger({ tabIds, windowId: window!.id! });
     return window!;
   },
+  // @ts-expect-error the type expects an overload
+  async update(
+    tabIdOrUpdateInfo: number | undefined | Tabs.UpdateUpdatePropertiesType,
+    optionalUpdateInfo: Tabs.UpdateUpdatePropertiesType | never,
+  ) {
```

Comment:
note: TS was complaining here. There is an overload https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/update#syntax, as you can pass one or two parameters. I opted to add the type annotation for all of the possible combinations.